modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,94 @@
+// tag::profileLinkingContextForUsers[]
+[[profile-linking-context-users]]
+= Linking Your Anypoint Platform Profiles
+
+When you log in to an Anypoint Platform using an email address that is associated with more than one profile in the same organization, you are prompted to link your profiles together. Your options for linking profiles vary based on whether the profiles use the Anypoint identity provider (IdP) or an external IdP that uses single sign-on (SSO). Anypoint Platform supports:
+
+* Linking an SSO profile to a local profile (login managed by Anypoint IdP)
+* Linking an SSO profile to another SSO profile that uses a different IdP to log in
+* Linking an SSO profile to multiple other profiles that each use different IdPs to log in
+
+Anypoint Platform does not support linking multiple profiles that use the same IdP. You can link up to 25 user profiles.
+
+// end::profileLinkingContextForUsers[]
+
+// tag::profileLinkingForSSOtoLocal[]
+[[sso-to-local]]
+= Linking an SSO Profile to a Local Profile
+
+When you attempt to log in using SSO and there is another user that shares the same email address but uses Anypoint Platform credentials to log in, you are prompted to link the profiles.
+
+. In the Anypoint Platform login page, click *Sign in with <your identity provider>*.
+. If prompted, enter your credentials for your IdP. +
+If your SSO credentials share an email address with an profile that is managed by the Anypoint IdP, you are prompted to log in. If you are not prompted, your local profile uses a different email address than your SSO profile.
++
+. In the *Link your Account* page, enter your credentials for your local Anypoint Platform profile.
+. Click *Link Accounts*.
++
+You can now view your linked profiles in the *Identity Providers* section of your user profile.
+
+// end::profileLinkingForSSOtoLocal[]
+
+// tag::profileLinkingSSOtoSSO[]
+[[sso-to-sso]]
+= Linking an Existing SSO Profile to Another SSO Profile
```

Comment:
Can we change the template from "linking profile to profile" to "linking profile to account". Our account model is like one account will have multiple IdP profiles. Well, it also makes sense to say "linking a profile to one / other profiles" . Let me think again

Reply:
The feature is mainly about linking a SSO profile to an existing account. Functionally, we can't actively link something existing to something else; rather, we link something new to something existing. I like the first section name "linking .. to a local profile" (I prefer local account here). 

What about following the same pattern for the other 2 section title? Say, "Linking a SSO profile to existing sso account"

Reply:
```suggestion
= Linking an Existing SSO Profile to Another SSO Profile
```